### PR TITLE
[WIP] clientconfig: Support for clouds-public.yaml

### PR DESCRIPTION
**Pull request by simonre**
Addresses #33

### CODE REVIEW

File: openstack/clientconfig/requests.go [hunk 1]
```diff
@@ -56,16 +57,26 @@ func LoadYAML() (map[string]Cloud, error) {
...
+
 	return clouds.Clouds, nil
 }
 
-// GetCloudFromYAML will return a cloud entry from a clouds.yaml file.
+// GetCloudFromYAML will return a cloud entry from a the yaml files used to specify the Clouds.
 func GetCloudFromYAML(opts *ClientOpts) (*Cloud, error) {
-	clouds, err := LoadYAML()
+
+	privateClouds, err := LoadYAML("clouds.yaml")
```
Comment (simonre) [inline openstack/clientconfig/requests.go]: I think it makes most sense to handle the different files and merge the clouds in this function even if that means changing the signature of LoadYAML to take a filename as well.

**Comment by simonre** [comment]
@jtopjian this is ready for review.

**Comment by simonre** [comment]
@jtopjian Actually, hold off on the review, after reviewing the openstack source code it seems like I misunderstood the purpose of clouds-public.yaml. I'll fix this pull request and update you.

**Comment by jtopjian** [comment]
@simonre No problem at all. :)

**Comment by simonre** [comment]
Closing in favour of #51

**Pull request closed without merging**